img/t/tori.fi.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-153 -46 652 652" xmlns:v="https://vecta.io/nano"><path d="M468.1 163.7c-.2 19.2-15.9 34.6-35.1 34.5s-34.7-15.7-34.7-34.9 15.6-34.8 34.8-34.8c9.3-.1 18.3 3.6 24.9 10.3 6.6 6.5 10.2 15.5 10.1 24.9h0zM138 222.8c-58.4 0-101.4 42.3-101.4 99s43 99 101.4 99c58.6 0 101.8-42.3 101.8-99s-43.2-99-101.8-99h0zm0 47.2c28.5 0 50 21.4 50 51.8 0 30.5-21.5 51.8-50 51.8s-49.8-21.4-49.8-51.8c0-30.5 21.3-51.8 49.8-51.8h0zm222-47.2c-23.1 0-41.1 8-52.9 25.6v-20.6H258V416h50.6V313.5c0-28 21.8-40.6 47.9-40.6 21.8 0 29.9 4.3 29.9 4.3v-49.3c-8.4-3.5-17.4-5.2-26.4-5.1h0zm98.7 5h-50.6V416h50.6V227.8zm-435.7 0h-68v-50.7h-50.6v50.7h-36.3v45h36.3v72.8c0 48.7 23.9 75.1 70.4 75.1a106.31 106.31 0 0 0 63.4-20.6L17 362.2c-10.3 6.2-21.8 11.4-33.8 11.4-15.7 0-28.1-8.5-28.1-32v-68.8H23v-45zm0 0" fill="#f94f55"/></svg>
```

Comment:
vecta.io/nano isn't the greatest when it comes to... vectors 😁
I recommend you run the files through SVGO or [SVGOMG](https://jakearchibald.github.io/svgomg/) if you want to minify vector graphics.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 292.3"><path fill="#f94f55" d="M600 35c-.1 19.11-15.8 34.8-35 34.7-19.2-.1-34.8-15.5-34.8-34.7 0-19.2 13.27-35 34.8-35 21.53 0 35.1 15.88 35 35zM269.9 94.3c-58.4 0-101.4 42.3-101.4 99s43 99 101.4 99c58.6 0 101.8-42.3 101.8-99s-43.2-99-101.8-99zm0 47.2c28.5 0 50 21.4 50 51.8 0 30.5-21.5 51.8-50 51.8s-49.8-21.4-49.8-51.8c0-30.5 21.3-51.8 49.8-51.8zm222-47.2c-23.1 0-41.1 8-52.9 25.6V99.3h-49.1v188.2h50.6V185c0-28 21.8-40.6 47.9-40.6 21.8 0 29.9 4.3 29.9 4.3V99.4c-8.4-3.5-17.4-5.2-26.4-5.1zm98.7 5H540v188.2h50.6zm-435.7 0h-68V48.6H36.3v50.7H0v45h36.3v72.8c0 48.7 23.9 75.1 70.4 75.1 22.8.1 45.02-7.11 63.4-20.6l-21.2-37.9c-10.3 6.2-21.8 11.4-33.8 11.4-15.7 0-28.1-8.5-28.1-32v-68.8h67.9z"/></svg>
```